tests/gui/test_dialog_model.py

```diff
@@ -271,7 +281,7 @@ def test_load_configuration_except_true(dialog_model: DialogModel, mocker) -> No
 def test_check_configuration(
     dialog_model: DialogModel,
     mock_branch: MagicMock,
-    mock_reach: MagicMock,
+    mock_areach: MagicMock,
     mock_reference_files: Dict[float, str],
     mock_measure_files: Dict[float, str],
```

Comment:
I expect that these should be filenameDict as well.

Reply:
@ralph-peelen-deltares we need to check this.

Reply:
Will update

Reply:
done